### PR TITLE
Add bash script to help fork a release branch

### PR DESCRIPTION
**Pull request by simonbaird**

### CODE REVIEW

File: hack/cut-release.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+BRANCH_NAME="redhat-${RELEASE_NAME}"
+RHTAP_APPLICATION_NAME="ec-${RELEASE_NAME/./}"
+
+echo Release name: $RELEASE_NAME
+echo Release branch name: $BRANCH_NAME
+echo RHTAP application name: $RHTAP_APPLICATION_NAME
+
+echo "Creating branch"
+git checkout -b $BRANCH_NAME main
```
Comment (joejstuart) [inline hack/cut-release.sh:44]: Do we need to make sure we have the latest upstream rev?  `git fetch upstream` and `git checkout -b $BRANCH_NAME upstream/main`?

**Review by zregvart — APPROVED** [review]
Looks okay to me

File: hack/cut-release.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+fi
+
+RELEASE_NAME=${1:-""}
+
+if [[ $RELEASE_NAME == "" ]]; then
+  echo "Please provide a release name, e.g. v0.1-preview, or v1.1"
+  exit 1
+fi
+
+BRANCH_NAME="redhat-${RELEASE_NAME}"
```
[the remark this answers is not in the document]
Comment (simonbaird) [reply hack/cut-release.sh:36]: I think it communicates to a random person looking at our repo what the branch is for. But other than that, not much.

[the remark this answers is not in the document]
Comment (simonbaird) [reply hack/cut-release.sh:36]: How about `release-*` instead?

**Comment by simonbaird** [comment]
This is no good actually. The component name is significant, since the pipeline definition file has a component name in it. So we have to copy the pipeline rather than modify it in place.

File: hack/cut-release.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (simonbaird) [reply hack/cut-release.sh]: In the new version it's just `v0.1-whatever`, no prefix at all.

**Review by joejstuart — APPROVED** [review]

**Pull request merged**